tests/test_resources.py

```diff
@@ -49,5 +49,77 @@ def test_httpverb(self):
             response = method('foo')
             assert response.success
 
+    def test_pagination(self):
+      self.client.url = 'mock://api.com/{param}{?per_page}'
```

Comment:
I don't think all (or any? not sure) of the paginated resources on the GitHub API actually advertise the `per_page` param (e.g. `public_gists_url` in the API root) so we should test the case where the URL doesn't have `per_page`.